pkg/db/db.go

```diff
@@ -141,12 +148,18 @@ func (dbc Config) forEach(rootBucket, nestedBucket string) (value map[string][]b
 				continue
 			}
 
+			source, _ := dbc.getDataSource(tx, r)
```

Comment:
It seems the error will be skipped. maybe we'll print it to logs?

Reply:
Added. https://github.com/aquasecurity/trivy-db/pull/183/commits/17ba4e367997fdea18270948f8b996c028a58d38

Reply:
approved. that's great!